html/captive-portal/lib/captiveportal/PacketFence/Model/Portal/Session.pm

```diff
@@ -110,8 +110,7 @@ sub ACCEPT_CONTEXT {
     my $options;
     my $mgmt_ip = $management_network->{'Tvip'} || $management_network->{'Tip'} if $management_network;
     my $destination_url = $request->param('destination_url');
-
-    if( defined ( my $last_uri = $r->pnotes('last_uri') )) {
+    if(  ref($r) ne 'GLOB' && ref($r) ne 'FileHandle' && defined ( my $last_uri = $r->pnotes('last_uri') )) {
```

Comment:
Could you please explain this line with a comment.

Also what does this have to do with the last_uri ?